Header                  header
//  Radar State & Config
Conti_radar_state       radar_state
Conti_radar_config      radar_config
Conti_filter_config     filter_config

// For objs output mode 
Conti_obj_status        objs_status 
Conti_obj_general[]     objs_general
Conti_obj_quality[]     objs_quality
Conti_obj_extended[]    objs_extended
Conti_obj_warning[]     objs_warning

// For cluster output mode 
Conti_cluster_status    cluster_status
Conti_cluster_general[] cluster_general
Conti_cluster_quality[] cluster_quality

// Motion Input
Conti_motion_rx         motion_rx


